src/azure-cli-core/azure/cli/core/extension/dynamic_install.py

```diff
@@ -110,10 +110,9 @@ def _get_extension_run_after_dynamic_install_config(cli_ctx):
 
 
 def _get_extension_allow_preview_install_config(cli_ctx):
-    default_value = True
+    default_value = False
     if cli_ctx and cli_ctx.config.get('extension', 'dynamic_install_allow_preview', None) is None:
-        logger.warning("Preview version of extension is enabled by default for extension installation now. "
-                       "Will be disabled in future release. ")
+        logger.warning("Preview version of extension is disabled by default for extension installation. ")
         logger.warning("Please run 'az config set extension.dynamic_install_allow_preview=true or false' "
                        "to config it specifically. ")
```

Comment:
Better to mention this config in `allow_preview` argument help msg as well

Reply:
`extension.dynamic_install_allow_preview` from config is designed for dynamic install.
`allow-preview` works specifically under `az extension add/update` and `az upgrade`.